src/LoggerAbstractServiceFactory.php

```diff
@@ -26,10 +26,18 @@ class LoggerAbstractServiceFactory implements AbstractFactoryInterface
      *
      * @var string
      */
-    protected $configKey = 'log';
+    protected $configKey;
+
+    public function __construct(string $configKey = 'log')
+    {
+        $this->configKey = $configKey;
+    }
 
     /**
-     * {@inheritDoc}
```

Comment:
Not sure why these got changed, but no biggie

Reply:
@Ocramius my editor suggested the change, because the interface docblock doesn't specify the exceptions - is it better  not make changes like this?

Reply:
It's not a problem, just noticed it and commented, but don't worry about it :)